com.zsmartsystems.zigbee.dongle.ember/src/test/java/com/zsmartsystems/zigbee/dongle/ember/internal/spi/SpiFrameHandlerTest.java

```diff
@@ -277,25 +278,36 @@ public void testConnect() {
     }
 
     @Test
-    public void testSendEzspFrame() {
+    public void testSendEzspFrame() throws NoSuchFieldException, IllegalAccessException {
         portOutData = new ArrayList<>();
         SpiFrameHandler handler = new SpiFrameHandler(Mockito.mock(EzspFrameHandler.class));
         handler.start(new TestPort(Mockito.mock(InputStream.class), Mockito.mock(OutputStream.class)));
 
+        final int sequenceNumber = 15;
+        setNextSequenceNumber(handler, sequenceNumber);
```

Comment:
You might want to use the ```TestUtilities``` class - it has a helper to set private fields. Since I fear in future this might all need to change, using the single method is probably useful.